Blockzilla/AboutViewController.swift

```diff
@@ -159,15 +174,41 @@ private class AboutHeaderView: UIView {
         aboutParagraph.textColor = UIConstants.colors.defaultFont
         aboutParagraph.font = UIConstants.fonts.aboutText
         aboutParagraph.numberOfLines = 0
-        addSubview(aboutParagraph)
+        return aboutParagraph
+    }()
 
+    private lazy var learnMoreButton : UIButton = {
         let learnMoreButton = UIButton()
         learnMoreButton.setTitle(UIConstants.strings.aboutLearnMoreButton, for: .normal)
         learnMoreButton.setTitleColor(UIConstants.colors.settingsLink, for: .normal)
         learnMoreButton.setTitleColor(UIConstants.colors.buttonHighlight, for: .highlighted)
         learnMoreButton.titleLabel?.font = UIConstants.fonts.aboutText
         learnMoreButton.addTarget(self, action: #selector(didPressLearnMore), for: .touchUpInside)
+        return learnMoreButton
+    }()
+
+    init() {
+        super.init(frame: CGRect.zero)
+        addSubviews()
+        configureConstraints()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    @objc private func didPressLearnMore() {
+        delegate?.aboutHeaderViewDidPressLearnMore(self)
+    }
+
+    private func addSubviews() {
+        addSubview(logo)
+        addSubview(aboutParagraph)
         addSubview(learnMoreButton)
+    }
+
+    private func configureConstraints() {
+        translatesAutoresizingMaskIntoConstraints = false
```

Comment:
I think SnapKit will automatically apply this to any views that get constrained

Reply:
You are absolutely correct. I have removed this line.